plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -331,6 +330,33 @@ public void testDropNonEmptySchemaWithTable()
         assertUpdate("DROP SCHEMA " + schemaName);
     }
 
+    @Override
+    public void testRenameColumn()
+    {
+        // Override because the connector doesn't support renaming columns with 'none' column mapping
+        // There are some tests in in io.trino.tests.product.deltalake.TestDeltaLakeColumnMappingMode
+        assertThatThrownBy(super::testRenameColumn)
```

Comment:
It would be beneficial to have https://github.com/trinodb/trino/pull/17565 so that we can properly create such tables.
Alternatively, `register_table` would be also an option (not my prefferred option).